### ISSUE TITLE
Docx Writer: inherit footnotes settings from reference-doc

### ISSUE THREAD

**Issue by ilcpm** [jgm/pandoc#7292]
In Chinese, we usually set first line align for the "normal text"

but this will result in the uncorrectable align of the footnotes headline (the line is also set with the style "normal text").

![image](https://user-images.githubusercontent.com/36830343/118393836-ba6afc80-b673-11eb-879c-04e835e20fbd.png)

![image](https://user-images.githubusercontent.com/36830343/118393964-67de1000-b674-11eb-911f-a3a7311f1cad.png)

To fix this, will set the separate line in file `footnotes.xml`.

![image](https://user-images.githubusercontent.com/36830343/118394035-e175fe00-b674-11eb-8d07-6f729aa2acf6.png)

I hope pandoc can also inherit these tags as #7240

**Comment by jgm** [comment]
Would putting a special style on the footnote and continuation separators solve the problem?

**Comment by ilcpm** [comment]
> Would putting a special style on the footnote and continuation separators solve the problem?


I set style "noindent" to it in reference.docx, but the style setting is in `footnotes.xml`, so the result won't inherit it, and still use style "normal text"

reference:

![image](https://user-images.githubusercontent.com/36830343/118404000-accd6b00-b6a3-11eb-98c3-e76224d6167c.png)

![image](https://user-images.githubusercontent.com/36830343/118404156-367d3880-b6a4-11eb-91e0-85949ff6838c.png)

result:

![image](https://user-images.githubusercontent.com/36830343/118404074-003fb900-b6a4-11eb-8e13-61819d59ace2.png)

![image](https://user-images.githubusercontent.com/36830343/118404136-26655900-b6a4-11eb-90f6-4c2fad1d8c93.png)

**Comment by ilcpm** [comment]
https://user-images.githubusercontent.com/36830343/118404683-82c97800-b6a6-11eb-847d-2783a6f1f0ab.mp4

I record a video to show you how to change the separate line to help you test this

**Comment by jgm** [comment]
But if we hard coded a special style in footnotes.xml that you could modify in styles.xml, that would solve the problem , right?

**Comment by jgm** [comment]
See #7087 -- is this the same issue?

**Comment by ilcpm** [comment]
> See #7087 -- is this the same issue?

Yes, it's true!

> But if we hard coded a special style in footnotes.xml that you could modify in styles.xml, that would solve the problem , right?

Set a special style can solve the problem of our project.

But, as seen in the tags, the separator actually is a paragraph, so it means that we can set anything (such as some text) as the separator.

So, the better choice is to inherit the tags with properties `w:type` as mentioned in [this](https://github.com/jgm/pandoc/issues/7087#issuecomment-808861314) reply

**Comment by jgm** [comment]
Right.  Incorporating elements from footnotes.xml from the reference.docx is going to be trickier to do without corruption, but we can probably do it.
